filebeat/module/tidb/_meta/docs.asciidoc

```diff
@@ -0,0 +1,215 @@
+    enabled: true
+-----
+
+* In the +filebeat.yml+ file
+
+["source","yaml",subs="attributes"]
+-----
+filebeat.modules:
+- module: tidb
+  slowlog:
+    enabled: true
+  tidb:
+    enabled: true
+  pd:
+    enabled: true
+  tikv:
+    enabled: true
+  tiflash:
+    enabled: true
+  binlog:
+    enabled: true
+  ticdc:
+    enabled: true
+  tispark:
+    enabled: true
+-----
+
+To specify the same settings at the command line, you use:
+
+["source","sh",subs="attributes"]
+
+-----
+-M "tidb.pd.enabled=true" -M "tidb.slowlog.enabled=true"
+-----
+
+:fileset_ex: slowlog
+
+[float]
+=== Override default settings
+
+[float]
+====  Deployment environment of the TiDB cluster
+
+The TiDB module assume TiDB cluster is deployed by TiUP. If you deploy the cluster in Kubernetes with TiDB operator, override the default environment:
+
+["source","yaml",subs="attributes"]
+-----
+filebeat.modules:
+- module: tidb
+  slowlog:
+    enabled: true
+    var:
+      deployment: container
+  tidb:
+    enabled: true
+    var:
+      deployment: container
+  pd:
+    enabled: true
+    var:
+      deployment: container
+  tikv:
+    enabled: true
+    var:
+      deployment: container
+  tiflash:
+    enabled: true
+    var:
+      deployment: container
+  binlog:
+    enabled: true
+    var:
+      deployment: container
+  ticdc:
+    enabled: true
+    var:
+      deployment: container
+  tispark:
+    enabled: true
+    var:
+      deployment: container
+-----
+
+[float]
+==== Paths and files
+
+The TiDB module come with two types of default paths:
+
+* Log Paths generated by TiUP.
+* Log Paths defined by TiDB operator, in the Kubernetes environment.
+
+The module also defines a default regex rule to exclude log files.
+
+The following example shows how to override those default paths and default excluded files:
+["source","yaml",subs="attributes"]
+-----
+filebeat.modules:
+- module: tidb
+  slowlog:
+    enabled: true
+    var:
+      paths: ["/tidb-deploy/tidb-4000/log/*slowlog*.log"]
+      exclude_files: [".gz$"]
+  tidb:
+    enabled: true
+    var:
+      paths: ["/tidb-deploy/tidb-4000/log/*.log"]
+      exclude_files: [".gz$", ".*slowlog.*\.log$"]
+  pd:
+    enabled: true
+    var:
+      paths: ["/tidb-deploy/pd-2379/log/*.log"]
+      exclude_files: [".gz$"]
+  tikv:
+    enabled: true
+    var:
+      paths: ["/tidb-deploy/tikv-20160/log/*.log"]
+      exclude_files: [".gz$"]
+  tiflash:
+    enabled: true
+    var:
+      paths: ["/tidb-deploy/tiflash-9000/log/*.log"]
+      exclude_files: [".gz$"]
+  binlog:
+    enabled: true
+    var:
+      paths: ["/tidb-deploy/pump-8250/log/*.log","/tidb-deploy/drainer-8249/log/*.log"]
+      exclude_files: [".gz$"]
+  ticdc:
+    enabled: true
+    var:
+      paths: ["/tidb-deploy/cdc-8300/log/*.log"]
+      exclude_files: [".gz$"]
+  tispark:
+    enabled: true
+    var:
+      paths: ["/tidb-deploy/tispark-master-7077/log/*.log","/tidb-deploy/tispark-worker-7078/log/*.log"]
+      exclude_files: [".gz$"]
```

Comment:
Why should we explicitly exclude `.gz`? Seems that it is not matched with `*.log`.

Reply:
Fixed.